src/OrchardCore.Modules/Orchard.Autoroute/Drivers/AutoroutePartDisplay.cs

```diff
@@ -61,14 +65,16 @@ public override IDisplayResult Edit(AutoroutePart autoroutePart)
         public override async Task<IDisplayResult> UpdateAsync(AutoroutePart model, IUpdateModel updater)
         {
             await updater.TryUpdateModelAsync(model, Prefix, t => t.Path);
-            
+
             var httpContext = _httpContextAccessor.HttpContext;
-            
             if (httpContext != null && await _authorizationService.AuthorizeAsync(httpContext.User, Permissions.SetHomepage))
             {
                 await updater.TryUpdateModelAsync(model, Prefix, t => t.SetHomepage);
             }
 
+            var validationErrorKeyPrefix = string.IsNullOrEmpty(Prefix) ? string.Empty : Prefix + ".";
```

Comment:
I have added `AddModelError` method that takes `prefix` as a parameter. It is in the `Microsoft.AspNetCore.Mvc.Modules.ModelBinding` namespace, the same namespace where `IUpdateModel` is.

If it is ok, I will also update other fields (probably it will be better to do it in a separate PR)